its/src/test/java/com/sonar/it/scanner/msbuild/utils/TestUtils.java

```diff
@@ -464,8 +462,9 @@ private static String extractCeTaskId(BuildResult buildResult) {
   }
 
   private static List<String> extractCeTaskIds(BuildResult buildResult) {
+    // INFO: More about the report processing at http://127.0.0.1:53395/api/ce/task?id=0f639b4c-6421-4620-81d0-eac0f5759f06
```

Comment:
I'm not sure I understand the comment, is the task id always the same? Isn't a generated GUID?

Reply:
It's a sample line from the logs, to see how the data looks like

Reply:
Got it. Mentioning that this is an excerpt from the logs would help.